### ISSUE TITLE
Misleading argument

### ISSUE THREAD

**Issue by albhasan** [e-sensing/sits#137]
The function `sits::sits_classify_raster` has the argument `file`, which  is actually a prefix instead of a path to a file. This may mislead users. I suggest renaming this argument as `file_prefix`

**Comment by pedro-andrade-inpe** [comment]
The documentation says that it is a prefix:

```
@param  file            File name prefix to store the output. For each time interval, one file will be created.
```

But I think it is interesting to update the name of the argument.

**Comment by gilbertocamara** [comment]
This issue has been solved in version 2.08.1 - the argument is now called "out_prefix".
